### DISCUSSION TITLE
8월 21일 중간점검 피드백

### DISCUSSION THREAD

**Discussion by Oleevia01** [CodeMedics/Harulog-backend#57] [Announcements]
1. 테스트코드 DisplayName  
- 어떤 상황에 대해 테스트 코드 작성 및 진행되었는지 자세하게 DisplayName 작성하기
- (최종)테스트 결과 보고서 엑셀표로 정리  DisplayName + 최종 결과 정리

2. 노력은 코드로 증명할 것(개발자로서 반영되도록 코드 중점으로 정리 및 반영)

3. 리팩토링이 필요하기 때문에 테스트코드를 작성하는 것임

4. READ ME 아키텍쳐 관련
- 초반부터 리드미에 아키텍쳐 설계하는 경우는 SI
- 서비스의 경우 개발하면서 변경될 수 있어서 유연하게 가져갈 것 

5. 컨트롤러 없이 서비스부터 작성하는 이유
- 로직을 탄탄하게 만들기 위함

**Comment by jeongmino1** [comment]
깔끔한 정리 감사합니다
